chrome/views/popup.pug

```diff
@@ -5,7 +5,7 @@ html
     meta(charset='UTF-8')
     title Heutagogy
     style.
-      body { width: 275px; height: 45px; }
+      body { width: 275px; }
```

Comment:
Why removing height?

Reply:
45px is about a height of a single toggle on my screen, so I only see a single input field at a time. (This is even more inconvenient when we draw a login screen.) 

By removing fixed height here we let the form control its own height as needed.